7.1/Dockerfile

```diff
@@ -10,7 +10,6 @@ RUN apt-get -qq update > /dev/null && DEBIAN_FRONTEND=noninteractive apt-get -qq
     ca-certificates \
     nginx \
     wget \
-    git \
```

Comment:
Yeah let's add git to other versions too, actually git is mostly added in inherited images, but it make sense to shit it by default :+1: